From a93bc8503649028852402c61bdda531af2872d0e Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Thu, 15 Oct 2020 09:15:06 +0200
Subject: [PATCH] src/build-data/arch/arm64.txt: add aarch64_be

Allow the user to set cpu value to aarch64_be.
Endianness will be correctly guessed by choose_endian function because
this alias ends with "be"

Fixes:
 - http://autobuild.buildroot.org/results/69ebf03c59b2af4140a39bc26f17d0396b6ec15d

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Retrieved from:
https://github.com/randombit/botan/commit/a93bc8503649028852402c61bdda531af2872d0e]
---
 src/build-data/arch/arm64.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/build-data/arch/arm64.txt b/src/build-data/arch/arm64.txt
index 056c5318ff..5205295125 100644
--- a/src/build-data/arch/arm64.txt
+++ b/src/build-data/arch/arm64.txt
@@ -5,6 +5,7 @@ family arm
 
 <aliases>
 aarch64
+aarch64_be
 armv8
 armv8-a
 </aliases>
